### PR TITLE
MM-10846 Make link previews configurable

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

File: mattermost-helm/values.yaml [hunk 1]
```diff
@@ -29,6 +29,8 @@ global:
         # Hosted push notification service. Requires an enterprise license.
         # More details at https://docs.mattermost.com/mobile/mobile-hpns.html.
         useHPNS: false
+    linkPreviews:
+      enabled: false
```
Comment (cpanato) [inline mattermost-helm/values.yaml]: maybe just add `linkPreviews: false` instead of having another level

Comment (jwilander) [reply mattermost-helm/values.yaml]: That's what we do with all the others, I'd rather be consistent: https://github.com/mattermost/mattermost-kubernetes/blob/6410af8588d4692f021211f3ce9686b46890a5ac/mattermost-helm/values.yaml#L12

Comment (cpanato) [reply mattermost-helm/values.yaml]: hummm, ok
but we should rewrite thos whole section and add some config level, like the mm-te. but lets do that in a followup

Comment (jwilander) [reply mattermost-helm/values.yaml]: Agreed, the whole thing needs a rewrite

**Pull request merged**